frontend/app/components/shelterDetail.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Col, Media } from 'react-bootstrap'
+import email from '../../static/email.png'
+import phone from '../../static/phone.png'
+import location from '../../static/location.png'
+
+class ShelterDetail extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  fillEmptyValues() {
```

Comment:
Rather than replace the keys with `false` it would be cleaner to simply return an array with those items removed altogether, but if this works for your purposes then it's good enough.